packages/rest-explorer/src/__tests__/acceptance/rest-explorer.acceptance.ts

```diff
@@ -36,11 +36,12 @@ describe('API Explorer (acceptance)', () => {
         .expect(/<title>LoopBack API Explorer/);
     });
 
-    it('redirects from "/explorer" to "/explorer/"', async () => {
+    it('exposes API Explorer at "/explorer"', async () => {
```

Comment:
This redirect is important because the way how relative paths are resolved in the browser. When the SPA UI is served from `/explorer`, a path like `src="swagger-ui.js"` is resolved to `/swagger-ui.js` instead of `/explorer/swagger-ui.js`.

I don't see how your patch is fixing this problem.

Can you revert the changes in REST Explorer please?

Reply:
Instead of depending on redirect, I updated `index.html.ejs` to honor `explorerPath` and it works now for `http://localhost:3000/explorer`.